nml/ast/alt_sprites.py

```diff
@@ -30,8 +30,10 @@
     "ZOOM_LEVEL_OUT_4X": 4,
     "ZOOM_LEVEL_OUT_8X": 5,
 }
+allow_extra_zoom = True
 
 bit_depths = {"BIT_DEPTH_8BPP": 8, "BIT_DEPTH_32BPP": 32.0}
+allow_32pbb = True
```

Comment:
```suggestion
allow_32bpp = True
```
And everywhere where it was transposed I guess.